apollo/handlers/websocket.py

```diff
@@ -19,3 +25,11 @@ async def shell(
         session=session, authorization=websocket.headers['authorization'])
     await WebSocketManager().add_and_connect_websocket(
         client_id=client_id, websocket=websocket)
+
+
+@router.get('/ws/{websocket_id}/close', permission='websocket.close')
+async def close_websocket_connection(websocket_id: uuid.UUID):
+    try:
+        await WebSocketManager().close_and_remove_connection(websocket_id)
```

Comment:
See comment I've made a while back on the ticket on how this is not really functionally complete.